src/runtime-rs/crates/resource/src/rootfs/block_rootfs.rs

```diff
@@ -67,8 +70,30 @@ impl BlockRootfs {
         let mut device_id: String = "".to_owned();
         if let DeviceType::Block(device) = device_info {
             storage.driver = device.config.driver_option;
-            storage.source = device.config.virt_path;
             device_id = device.device_id;
+
+            match block_driver.clone().as_str() {
```

Comment:
I think the `.clone()` is not required here and directly use `block_driver.as_str()` is fine.

Reply:
Hi, @amshinde @jodh-intel I have no other required changes request but this one.